api/_lib/script/build.ts

```diff
@@ -71,10 +71,16 @@ async function main() {
     }
 
     const fullDir = join(packagesDir, pkg);
-    const packageJsonRaw = await fs.readFile(
-      join(fullDir, 'package.json'),
-      'utf-8'
-    );
+    const packageJsonRaw = await fs
+      .readFile(join(fullDir, 'package.json'), 'utf-8')
+      .catch(() => null);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏can we log the error?